### DISCUSSION TITLE
css/schemes not being read after last update

### DISCUSSION THREAD

**Discussion by arahmitz** [nunocoracao/blowfish#1327] [Q&A]
Hey, I wonder - does anyone else have problems with the color schemes after the update? Seems like it started defaulting to the original blowfish color scheme no matter what you type in params - seems like they background color mask is working properly, but I can't get any additional colors to work.

**Comment by arahmitz** [comment]
Tested this on new project - seems like Tailwind colors aren't invoked by `colorScheme` anymore, seems like a bug.

**Comment by nunocoracao** [comment]
Thanks @arahmitz let's continue the discussion on https://github.com/nunocoracao/blowfish/issues/1329
